runtime/builtin_types_test.go

```diff
@@ -23,6 +23,32 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	f := NewRootFrame()
+	delattr := mustNotRaise(Builtins.GetItemString(NewRootFrame(), "delattr"))
+	fooType := newTestClass("Foo", []*Type{ObjectType}, newStringDict(map[string]*Object{}))
+	fooForDelAttr := newObject(fooType)
+	SetAttr(f, fooForDelAttr, NewStr("bar"), None)
+	fun := wrapFuncForTest(func(f *Frame, args ...*Object) (*Object, *BaseException) {
+		_, raised := delattr.Call(f, args, nil)
+		if raised != nil {
+			return nil, raised
+		}
+		return GetAttr(f, args[0], toStrUnsafe(args[1]), nil)
```

Comment:
I would use None (or something else) as a marker value so that GetAttr doesn't raise in the successful case. That will make the test cases a little more easy to understand (took me a minute to understand why all the test cases had wantExc).

I would also return the result of delattr along with the GetAttr result in a tuple (like in TestBuiltinSetAttr below) so that you can verify that delattr returns None.